redislock.go

```diff
@@ -265,11 +266,12 @@ func ExponentialBackoff(min, max time.Duration) RetryStrategy {
 }
 
 func (r *exponentialBackoff) NextBackoff() time.Duration {
-	r.cnt++
+	atomic.AddUint32(&r.cnt, 1)
```

Comment:
you can assign the result of that `cnt := 	atomic.AddUint32(&r.cnt, 1)`

Reply:
Updated.